IdentityCore/tests/MSIDVersion.m

```diff
@@ -51,4 +51,10 @@ + (NSString *)telemetryEventPrefix
     return @"Microsoft.Test.";
 }
 
++ (NSString *)aadApiVersion
+{
+    return nil;
```

Comment:
nit: should we provide something default for tests? e.g. @"test-api-version"?

Reply:
added